front/src/Dropdown/index.test.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Dropdown from ".";
+import renderer from "react-test-renderer";
+
+describe("Dropdown", () => {
+  it("can be changed if enabled", () => {
+    let value = "test";
+    const component = renderer.create(
+      <Dropdown
+        currentValue={value}
+        values={["test", "test2"]}
+        enabled
+        onChange={newValue => {
+          value = newValue;
+        }}
+      />
+    );
+    let tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+
+    // Check if we can trigger onChange
+    expect(value).toBe("test");
+    tree.props.onChange({ target: { value: "test2" } });
+    expect(value).toBe("test2");
+  });
+
+  it("cannot be changed when disabled", () => {
+    let value = "tea";
+    const component = renderer.create(
+      <Dropdown
+        currentValue={value}
+        values={["tea", "coffee", "water"]}
+        enabled={false}
+        onChange={newValue => {
+          value = newValue;
+        }}
+      />
+    );
+    let tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+
+    // Check if we can't trigger onChange
+    expect(value).toBe("tea");
+    tree.props.onChange({ target: { value: "coffee" } });
+    expect(value).toBe("tea");
+  });
+});
```

Comment:
Just matching a snapshot should be enough for testing this component. I dont think you have to test the `disabled` html attribute.